code/game/objects/items/weapons/implants/implantfluff.dm

```diff
@@ -0,0 +1,112 @@
+
+/**
+ * # Implant Fluff Datum
+ *
+ * Implant fluff is just lore about the implant that is accessed through the implantpad, you must attach
+ * one of these datums to the implant_data var on an implant for it to show it up.
+ */
+/datum/implant_fluff
+	var/name = "Unknown Implant"
+	var/life = "Not Specified"
+	var/notes = "No Notes"
```

Comment:
You might want to make sure everything in here has periods where they need it. Especially since these are all fluff.

Reply:
👍 will do